src/test/scala/ColumnTest.scala

```diff
@@ -0,0 +1,136 @@
+package CSVValidation
+import com.fasterxml.jackson.databind.{JsonNode, ObjectMapper}
+import com.fasterxml.jackson.databind.node.{
+  IntNode,
+  JsonNodeFactory,
+  ObjectNode,
+  TextNode
+}
+import org.scalatest.FunSuite
+
+import scala.collection.mutable
+import scala.collection.mutable.Map
+
+class ColumnTest extends FunSuite {
+  val objectMapper = new ObjectMapper()
+  test("should provide appropriate default values") {
+    val json =
+      """
+        |{
+        |"name":"countryCode"
+        |}"
+        |""".stripMargin
+
+    val jsonNode = objectMapper.readTree(json)
+    val column = Column.fromJson(
+      1,
+      jsonNode.asInstanceOf[ObjectNode],
+      "https://www.w3.org/",
+      "und",
+      Map[String, JsonNode]()
+    )
+
+    val datatypeDefaultValue: ObjectNode = JsonNodeFactory.instance
+      .objectNode()
+      .set("@id", new TextNode("http://www.w3.org/2001/XMLSchema#string"))
+
+    assert(column.name.get === "countryCode")
+    assert(column.datatype === datatypeDefaultValue)
+    assert(column.lang === "und")
+    assert(column.textDirection === "inherit")
+    assert(column.annotations === Map[String, JsonNode]())
+    assert(!column.virtual)
+    assert(column.number == 1)
+    assert(!column.ordered)
+    assert(!column.required)
+    assert(!column.suppressOutput)
+    assert(column.aboutUrl.isEmpty)
+    assert(column.default == "")
+    assert(column.propertyUrl.isEmpty)
+    assert(column.separator.isEmpty)
+    assert(column.titles.isEmpty)
+  }
+
+  test("should override default values") {
+    val json =
+      """
+        |{
+        |   "name": "countryCode",
+        |   "titles": "countryCode",
+        |   "propertyUrl": "http://www.geonames.org/ontology"
```

Comment:
If you really want to test this, you'll need to provide specific values for each of the properties in this test. I would say that would be worthwhile.